cloudify_scalelist/workflows.py

```diff
@@ -424,6 +425,7 @@ def _run_scale_settings(ctx, scale_settings, scalable_entity_properties,
                             )
                         )
             related = removed_and_related - removed
+            _wait_for_sent_tasks(ctx, graph)
```

Comment:
is this necessary

Reply:
yeah, i added the wait before every `_uninstall_instances` call, because `_uninstall_instances` clears the graph - so that's the only opportunity we have to wait for the tasks that have been already sent, before we effectively forget about them